tests/S3/UseArnRegion/ConfigurationProviderTest.php

```diff
@@ -99,6 +99,37 @@ public function testCreatesDefaultFromFallback()
         $this->assertSame($expected->toArray(), $result->toArray());
     }
 
+    public function testUsesIniWithUseAwsConfigFileTrue()
+    {
+        $dir = $this->clearEnv();
+        putenv(ConfigurationProvider::ENV_CONFIG_FILE . '=' . $dir . "/alt_config");
+        $expected = new Configuration(true);
+        file_put_contents($dir . '/config', $this->iniFile);
+        file_put_contents($dir . '/alt_config', $this->altIniFile);
```

Comment:
Is there any reason this test needs an alt_config?

Reply:
Yes, I removed the /config to clear that up.  The default values come in with no change if I just use the normal config file, so the test doesn't know for sure whether or not it actually used the file. 

 The other options are adding a new redundant configuration with a new name, or I could putting the altIniFile's contents in as the normal config file.  Let me know if you'd prefer I did it another way, I kept in the /alt_config file to be clear that we were using the input of the alt ini file instead of the standard one

Reply:
The way you changed it seems reasonable.